### PR TITLE
Updated details and text of all plugins 😋

### PR DESCRIPTION
**Pull request by NavaneethRenjith8282**
I've given some changes to details and texts of all plugins in Userge. I've taken 3-4 hours to make this change.
## Explanation of some changes
I've changed smiley icons to emojis. Because, messages in Telegram supports emojis, as in inline also.
_For example, I've changed_ :) _icon to_ 🙂 _emoji._
Then, I've made changes to some texts.
## Renamed some plugins...
- I've given a change to `voice-call` plugin.
I've changed it's name to `video-chat`.
Because, Telegram has changed renamed Voice Chat as **Video Chat**.
- I've changed `gdrive` to `google_drive`. Because, Google Drive, Google are trademarks of **Google LLC**. And, I don't like to show it as `GDrive`. That's why I made up this change.

And, there is another changes also. You can check it at the [Commits](https://github.com/UsergeTeam/Userge/pull/417/commits) tab.
## I'm hoping...
I'm hoping that you'll push this because, I've worked for 3-4 hours for this change.
### Note:
- I've changed only the names and texts of plugins, commands, and it's texts. I didn't changed any other thing.

### CODE REVIEW

File: userge/plugins/tools/ping.py [hunk 1]
```diff
@@ -19,4 +19,4 @@ async def pingme(message: Message):
     await message.edit('`Pong!`')
     end = datetime.now()
     m_s = (end - start).microseconds / 1000
-    await message.edit(f"**Pong!**\n`{m_s} ms`")
+    await message.edit(f"**Pong! I'm alive!**\n`{m_s} ms`")
```
Comment (Krishna-Singhal) [inline userge/plugins/tools/ping.py:22]: atleast dont change ping 😢😫

**Review by Krishna-Singhal — CHANGES_REQUESTED** [review]

File: userge/plugins/tools/ping.py (hunk 1 above)
Comment (NavaneethRenjith8282) [reply userge/plugins/tools/ping.py:22]: Ok, ok, ok.

Comment (NavaneethRenjith8282) [reply userge/plugins/tools/ping.py:22]: > atleast dont change ping 😢😫

Ok, ok, ok.
I've changed it to defaults.

Comment (NavaneethRenjith8282) [reply userge/plugins/tools/ping.py]: > atleast dont change ping 😢😫

Really, I've changed it as like `ping` in [Ultroid](https://github.com/TeamUltroid/Ultroid).
If it doesn't matter to be want, don't be worried. I've changed it.

**Comment by NavaneethRenjith8282** [comment]
> @NavaneethRenjith8282, Thanks for updating this PR.
> 
> * In the file [`userge/plugins/utils/weather.py`](https://github.com/UsergeTeam/Userge/blob/0c6f91ac2d972a25def59a1c0184ce3bdda4bfcb/userge/plugins/utils/weather.py):
> 
> > [Line 47:76](https://github.com/UsergeTeam/Userge/blob/0c6f91ac2d972a25def59a1c0184ce3bdda4bfcb/userge/plugins/utils/weather.py#L47): [W291](https://duckduckgo.com/?q=pep8%20W291) trailing whitespace
> 
> ##### Comment last updated at 2021-12-12 06:49:23 UTC

Lint Errors fixed due to recommendation of @Krishna-Singhal . [View message here...](https://t.me/usergeot/777691)

**Comment by Krishna-Singhal** [comment]
> > @NavaneethRenjith8282, Thanks for updating this PR.
> > 
> > * In the file [`userge/plugins/utils/weather.py`](https://github.com/UsergeTeam/Userge/blob/0c6f91ac2d972a25def59a1c0184ce3bdda4bfcb/userge/plugins/utils/weather.py):
> > 
> > > [Line 47:76](https://github.com/UsergeTeam/Userge/blob/0c6f91ac2d972a25def59a1c0184ce3bdda4bfcb/userge/plugins/utils/weather.py#L47): [W291](https://duckduckgo.com/?q=pep8%20W291) trailing whitespace
> > 
> > 
> > ##### Comment last updated at 2021-12-12 06:49:23 UTC
> 
> Lint Errors fixed due to recommendation of @Krishna-Singhal . [View message here...](https://t.me/usergeot/777691)

They are still there :(

**Comment by NavaneethRenjith8282** [comment]
> > Lint Errors fixed due to recommendation of @Krishna-Singhal . [View message here...](https://t.me/usergeot/777691)
> 
> They are still there :(

No problem, @Krishna-Singhal. You can still edit this because, I've given it's access “Allow edit access by maintainers”. So, you can edit it and merge it. 🙂

**Comment by Jigarvarma2005** [comment]
> I've given some changes to details and texts of all plugins in Userge. I've taken 3-4 hours to make this change.
> 
> ## Explanation of some changes
> I've changed smiley icons to emojis. Because, messages in Telegram supports emojis, as in inline also. _For example, I've changed_ :) _icon to_ 🙂 _emoji._ Then, I've made changes to some texts.
> 
> ## Renamed some plugins...
> * I've given a change to `voice-call` plugin.
>   I've changed it's name to `video-chat`.
>   Because, Telegram has changed renamed Voice Chat as **Video Chat**.
> * I've changed `gdrive` to `google_drive`. Because, Google Drive, Google are trademarks of **Google LLC**. And, I don't like to show it as `GDrive`. That's why I made up this change.
> 
> And, there is another changes also. You can check it at the [Commits](https://github.com/UsergeTeam/Userge/pull/417/commits) tab.
> 
> ## I'm hoping...
> I'm hoping that you'll push this because, I've worked for 3-4 hours for this change.
> 
> ### Note:
> * I've changed only the names and texts of plugins, commands, and it's texts. I didn't changed any other thing.

Try this for weather.py pep8 fix
👇👇
[Weather.py](https://katb.in/enudulerobo.py)

**Comment by NavaneethRenjith8282** [comment]
> Try this for weather.py pep8 fix 👇👇 [Weather.py](https://katb.in/enudulerobo.py)

**Thanks for helping me,** @Jigarvarma2005! I've **changed** the [commit](https://github.com/UsergeTeam/Userge/pull/417/commits/fae3cc5b8a0ec9163f7fd006a4737c44dd03bcc5) to the text you've given.

**Comment by Jigarvarma2005** [comment]
> > Try this for weather.py pep8 fix 👇👇 [Weather.py](https://katb.in/enudulerobo.py)
> 
> **Thanks for helping me,** @Jigarvarma2005! I've **changed** the [commit](https://github.com/UsergeTeam/Userge/pull/417/commits/fae3cc5b8a0ec9163f7fd006a4737c44dd03bcc5) to the text you've given.

fix deepsource too

**Comment by NavaneethRenjith8282** [comment]
> > > Try this for weather.py pep8 fix 👇👇 [Weather.py](https://katb.in/enudulerobo.py)
> > 
> > 
> > **Thanks for helping me,** @Jigarvarma2005! I've **changed** the [commit](https://github.com/UsergeTeam/Userge/pull/417/commits/fae3cc5b8a0ec9163f7fd006a4737c44dd03bcc5) to the text you've given.
> 
> fix deepsource too

I want to consider creating decoration with `@staticmethod` because, [DeepSource](https://deepsource.io/gh/UsergeTeam/Userge/) says to do it for perfomance improvements. 
Should I want to do [this](https://deepsource.io/gh/UsergeTeam/Userge/run/f9af76d6-0144-439a-9394-18ac613ae2f6/python/)?

**Comment by Jigarvarma2005** [comment]
> > > > Try this for weather.py pep8 fix 👇👇 [Weather.py](https://katb.in/enudulerobo.py)
> > > 
> > > 
> > > **Thanks for helping me,** @Jigarvarma2005! I've **changed** the [commit](https://github.com/UsergeTeam/Userge/pull/417/commits/fae3cc5b8a0ec9163f7fd006a4737c44dd03bcc5) to the text you've given.
> > 
> > 
> > fix deepsource too
> 
> I want to consider creating decoration with `@staticmethod` because, [DeepSource](https://deepsource.io/gh/UsergeTeam/Userge/) says to do it for perfomance improvements. 😕 It wants to be in this line:
> 
> ```
>   async def paste(self, ses: aiohttp.ClientSession,
> ```
> 
> Should I want to do [this](https://deepsource.io/gh/UsergeTeam/Userge/run/f9af76d6-0144-439a-9394-18ac613ae2f6/python/)?

try this for deepsource

[rss_feed.py](https://katb.in/egileqoyofi)

**Comment by Jigarvarma2005** [comment]
lol

**Comment by Harsh-br0** [comment]
He can just practice GitHub with this PR ig , already messed with code

**Comment by Jigarvarma2005** [comment]
> He can just practice GitHub with this PR ig , already messed with code

yeah

**Comment by NavaneethRenjith8282** [comment]
# Sorry...
I don't know why this issue need this much problems. Someone says in comment:

>> @Harsh-br0 :
>
> He can just practice GitHub with this PR ig , already messed with code

Yes. I can also do it.
But, there is still many problems in this pull request. So, maybe I think to **close this pull request**.
Don't worry, I won't close this now.
**Please wait until I get any ideas...**

**Comment by Harsh-br0** [comment]
> I don't know why this issue need this much problems

Not your fault too but you should control your curiosity like first get the knowledge before doing it and be eligible to go for it

**Comment by rking32** [comment]
@NavaneethRenjith8282 Sorry to say this. Apparently, you have messed up the code.

**Comment by NavaneethRenjith8282** [comment]
> @NavaneethRenjith8282 Sorry to say this. Apparently, you have messed up the code.
Sorry everyone. I'm closing now.

**Comment by NavaneethRenjith8282** [comment]
> # Sorry...
> I don't know why this issue need this much problems. Someone says in comment:
> 
> > > @Harsh-br0 :
> > 
> > 
> > He can just practice GitHub with this PR ig , already messed with code
> 
> Yes. I can also do it. But, there is still many problems in this pull request. So, maybe I think to **close this pull request**. Don't worry, I won't close this now. **Please wait until I get any ideas...**

So, **closing** this issue now.
**Thanks for helping me!!!*** ❤️❤️❤️

**Pull request closed without merging**